backend/.checkstyle

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <fileset-config file-format-version="1.2.0" simple-config="false" sync-formatter="false">
-  <local-check-config name="maven-checkstyle-plugin verify-style" location="file:/D:/Git/Contextproject-BeNine/checkstyle.xml" type="remote" description="maven-checkstyle-plugin configuration verify-style">
+  <local-check-config name="maven-checkstyle-plugin verify-style" location="file:/C:/Users/bryan/Documents/Git/Contextproject-BeNine/checkstyle.xml" type="remote" description="maven-checkstyle-plugin configuration verify-style">
```

Comment:
This file is removed from git in https://github.com/Kingdorian/Contextproject-BeNine/pull/122/commits/956ca842581d7ae6b751f9cd84299bae10317601